modules/ROOT/pages/anypoint-insights.adoc

```diff
@@ -0,0 +1,504 @@
+----
+
+|`entity`
+|`=`, `IN`
+|Filter by specific app or API deployments, for example:
+----
+entity IN ['99aea888-77de-6e06-5c55-d1234de56789', '09f99999-888d-7b77-6da5-bebd43210fc1']
+----
+
+|===
+
+`IN` captures exact matches to any comma-separated element in the array.
+
+//  H3: FILTER TIME PERIOD
+[[filter-by-time]]
+=== Filter Data by Time Period
+
+Use the time period selector (<<dashboard-overview, item 6>> in the example) to display data points collected for a given time period (such as the last 15 minutes or the last 24 hours). Your setting applies across all data in *Anypoint Insights* overview, including the *Entities List*.
+
+If you don't see data on the page, check that the selected time range is correct.
+
+//
+//H2: CHANGE ENVIRONMENT
+[[change-environment]]
+== Change the Environment
+
+Charts in Anypoint Insights provide metrics on a specific environment (such as Production or Sandbox). 
+
+To change current or default environment:
+
+. Open Anypoint Monitoring. 
+. Set a new environment:
+
+* To select an environment without changing the default, use the environment selector in Anypoint Monitoring, for example:
++
+image::insights-environment-selector.png["Environment selector in Anypoint Monitoring"]
+* To select a default environment:
+
+.. Click your profile icon (see <<profile, item 2>> in the example). 
+.. Select the name of your profile to open Access Management *Settings*.
+.. In *Settings*, select an environment as the default environment to view for the selected business group in Anypoint Monitoring, for example: 
++
+image::insights-default-selector.png["Default selector in Access Management"]
+.. Return to Anypoint Monitoring to check for the new business group setting.
++
+Refresh the page, if necessary. 
+
+//
+//H2: CHANGE BUSINESS GROUP
+[[change-bg]]
+== Change the Business Group
+
+Charts in Anypoint Insights display metrics for deployments in a specific business group. If you belong to more than one business group, you can change the selected group and the business group that Anypoint Monitoring selects by default.
+
+//do not change the order of numbers in the graphic or calloutlist
+[[profile]]
+image::insights-business-group-change.png["Business group selection options"]
+
+[calloutlist]
+. Business group selector 
+. User profile, for selecting defaults
+
+To change the business group setting: 
+
+. Open Anypoint Monitoring. 
+. Set a new business group:
+
+* To select a business group without changing the default, use the business group selector in Anypoint Insights. 
+* To select a default business group:
+
+.. Click your profile icon (see <<profile, item 2>> in the example). 
+.. Select the name of your profile to open Access Management *Settings*.
+.. In Access Management *Settings*, select a business group to use as the default, for example:
++
+image::insights-default-selector.png["Default selector in Access Management"]
+.. Return to Anypoint Monitoring to check for the new default setting.
```

Comment:
Consider using partials for 227 and 229.

Reply:
Adding to [W-17361464](https://gus.lightning.force.com/a07EE000026JsovYAC)

Reply:
Followup: 
* 227 isn't repeated anywhere at this time. 

* 229 is repeated, so an include is warranted.